### DISCUSSION TITLE
Solutes Accumulating at Boundaries in Solute Transport Model with Zero Flux

### DISCUSSION THREAD

**Discussion by Alvin-prc** [modflowpy/flopy#2065] [Q&A]
While setting up a solute transport model, I noticed that when my boundaries in the icbund array are set to zero flux (even just one boundary set to zero flux), the calculated solutes pile up at the boundaries, with a value of 1e30. I realize this must be an error, as there isn't that much solute in my system, and the transport couldn't possibly be so rapid. It's puzzling why this occurs. This issue doesn't occur when all icbund values are set to 1. Can anyone shed light on what might be causing this problem?

**Comment by RyanConway91** [comment]
Is there a mass balance issue(check .lst files)? Can the sink/sources balance out? If you are only using specified flux BCs (WEL, RCH) you likely have an unsolvable system. Try using CHDs instead maybe?
